grow/translators/google_sheets.py

```diff
@@ -54,13 +55,12 @@ def _download_sheet(self, spreadsheet_id, locale):
                 raise translator_errors.NotFoundError(
                     'Translation for {} not found.'.format(locale))
             raise
-
         return resp['values']
 
     def _download_content(self, stat):
         spreadsheet_id = stat.ident
         values = self._download_sheet(spreadsheet_id, stat.lang)
-        source_lang, lang = values.pop(0)
+        source_lang, lang, _ = values.pop(0)
```

Comment:
From what I can tell it does fine with more columns since we have to specifically request which columns we want (ex: A:C in this case). So additional columns should not matter, as long as the first three are ours.